src/drivers/queue_interop/Queue.php

```diff
@@ -0,0 +1,153 @@
+<?php
+
+namespace yii\queue\queue_interop;
+
+use Enqueue\AmqpTools\DelayStrategyAware;
+use Enqueue\AmqpTools\RabbitMqDelayPluginDelayStrategy;
+use Enqueue\AmqpTools\RabbitMqDlxDelayStrategy;
+use Interop\Amqp\AmqpContext;
+use Interop\Amqp\AmqpQueue;
+use Interop\Queue\PsrConnectionFactory;
+use Interop\Queue\PsrContext;
+use Interop\Queue\PsrProducer;
+use yii\base\NotSupportedException;
+use yii\queue\cli\Queue as CliQueue;
+
+class Queue extends CliQueue
+{
+    const RABBITMQ_DELAY_DLX = 'dlx';
```

Comment:
We tried to keep it as simple as possible, so everything in one class. 

we can move amqp interop specific stuff to separate class. Is it ok?